addons/automatic_updates.sh

```diff
@@ -38,8 +40,9 @@ else
 fi
 
 # Install automatic updates
-touch $VMLOGS/update.log
-crontab -u root -l | { cat; echo "0 $AUT_UPDATES_TIME * * 6 $SCRIPTS/update.sh minor >> $VMLOGS/update.log 2>&1"; } | crontab -u root -
+DATE='`date +%Y-%m-%d_%H:%M`'
+mkdir -p "$VMLOGS"/updates
+crontab -u root -l | { cat; echo "0 $AUT_UPDATES_TIME * * 6 $SCRIPTS/update.sh minor >> $VMLOGS/updates/update-${DATE}.log 2>&1"; } | crontab -u root -
```

Comment:
@enoch85 I didn't test this but it looks like all updates just get written to an update log file that is called e.g.
/var/log/nextcloud/updates/update-2021-05-20_22:16.log instead of getting written to /var/log/nextcloud/update.log.

Reply:
That will expand with the current date in the file name, not the date that it's being run.

This PR is 100% intentional. I will delete old logs in another PR as well.

Thanks for the pointers! 👍🏻

Reply:
yes, but your PR doesn't work as intended, as far as I can see.
Please see my suggestion.

Reply:
After thinking about it it might work excactly like my proposal.

Reply:
Have you tested it?

Reply:
No but I am able to read the code ;)

Reply:
OK, everything works as intended anyway. 

I had missed to espace the date variable correctly (did it in my testing but not in the final PR) but that's fixed now.

The intention here is to get a new file for each run which is how it works now.

By doing as you suggest the date variable would be expanded in the crontab filename directly, and always get the same date, to the same file - which is wrong.

Reply:
> The intention here is to get a new file for each run which is how it works now.

Yes, that is also what my proposal is doing

> By doing as you suggest the date variable would be expanded in the crontab filename directly, and always get the same date, to the same file - which is wrong.

I though this would be the mistake that you've done in your PR but obviously we've found two solutions to the same idea ;)
Concerning my proposal: By escaping the date command, every log gets its own log file.

Reply:
Tried that already, didn't work. Hence the ' ' in the date variable. 

Your output: 
/var/log/nextcloud/updates/update-$(2021-05-21_08:27).log 2>&1

What we want (current solution):
/var/log/nextcloud/updates/update-`date +\%Y-\%m-\%d_\%H:\%M`.log 2>&1

Reply:
OK, rest my case. Works without the back ticks.

Reply:
hm... the escaping should technically do it... strange